### PR TITLE
[CRIMRE-362] Remove Interests of Justice MAAT property

### PR DESCRIPTION
**Pull request by mseedat-moj**
## Description of change
Removes `interests_of_jusice` from MAAT JSON only, maintains for CrimeApplications, using Schema v1.0.0

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMRE-396

### CODE REVIEW

**Comment by zheileman** [comment]
There is an extraneous prometheus file floating around...

Also, I would go the other way around, first modify the schemas gem, and then use the new version in datastore, otherwise, once this is merged, there will be a mismatch between what we say in the MAAT schema we return, and what we are returning.

Confirm with MAAT they are ready for this removal.

**Review by zheileman — CHANGES_REQUESTED** [review]
There is an extraneous prometheus file floating around...

Also, I would go the other way around, first modify the schemas gem, and then use the new version in datastore, otherwise, once this is merged, there will be a mismatch between what we say in the MAAT schema we return, and what we are returning.

Confirm with MAAT they are ready for this removal.

File: app/api/datastore/entities/v1/base_application_entity.rb [hunk 1]
```diff
@@ -11,7 +11,6 @@ class BaseApplicationEntity < Grape::Entity
         expose :provider_details
         expose :client_details
         expose :case_details
-        expose :interests_of_justice
```
Comment (zheileman) [inline app/api/datastore/entities/v1/base_application_entity.rb:14]: I may feel inclined to leave the base entity untouched and just add to the maat entity:

`unexpose : interests_of_justice`

Feels to me more explicit that way. But functionally would be the same.

**Comment by mseedat-moj** [comment]
Closed because MAAT team have not yet removed `interests_of_jusice` field from their schema

**Pull request closed without merging**